{
  "schema_version": "1.6.3",
  "id": "USN-4851-1",
  "summary": "libsolv vulnerabilities",
  "details": "It was discovered that Libsolv incorrectly handled certain malformed input.\nAn attacker could use this issue to cause Libsolv to crash, resulting in a\ndenial of service.",
  "aliases": [],
  "related": [
    "CVE-2018-20532",
    "CVE-2018-20533",
    "CVE-2018-20534"
  ],
  "published": "2021-03-15T22:36:33.444145Z",
  "modified": "2021-03-15T22:36:33.444145Z",
  "affected": [
    {
      "package": {
        "ecosystem": "Ubuntu:Pro:16.04:LTS",
        "name": "libsolv",
        "purl": "pkg:deb/ubuntu/libsolv@0.6.11-1.1ubuntu0.1~esm1?arch=src?distro=esm-apps/xenial"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "0.6.11-1.1ubuntu0.1~esm1"
            }
          ]
        }
      ],
      "versions": [
        "0.6.11-1",
        "0.6.11-1build1",
        "0.6.11-1.1"
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "libsolv-doc": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv-perl": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv-perl-dbgsym": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv-tools": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv-tools-dbgsym": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv0": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv0-dbg": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv0-dbgsym": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv0-dev": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolv0-dev-dbgsym": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolvext0": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolvext0-dbg": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolvext0-dbgsym": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolvext0-dev": "0.6.11-1.1ubuntu0.1~esm1",
            "libsolvext0-dev-dbgsym": "0.6.11-1.1ubuntu0.1~esm1",
            "python-solv": "0.6.11-1.1ubuntu0.1~esm1",
            "python3-solv": "0.6.11-1.1ubuntu0.1~esm1"
          }
        ],
        "availability": "Available with Ubuntu Pro: https://ubuntu.com/pro"
      }
    },
    {
      "package": {
        "ecosystem": "Ubuntu:Pro:18.04:LTS",
        "name": "libsolv",
        "purl": "pkg:deb/ubuntu/libsolv@0.6.30-1ubuntu0.1~esm1?arch=src?distro=esm-apps/bionic"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "0.6.30-1ubuntu0.1~esm1"
            }
          ]
        }
      ],
      "versions": [
        "0.6.28-2build1",
        "0.6.30-1build1"
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "libsolv-doc": "0.6.30-1ubuntu0.1~esm1",
            "libsolv-perl": "0.6.30-1ubuntu0.1~esm1",
            "libsolv-perl-dbgsym": "0.6.30-1ubuntu0.1~esm1",
            "libsolv-tools": "0.6.30-1ubuntu0.1~esm1",
            "libsolv-tools-dbgsym": "0.6.30-1ubuntu0.1~esm1",
            "libsolv0": "0.6.30-1ubuntu0.1~esm1",
            "libsolv0-dbgsym": "0.6.30-1ubuntu0.1~esm1",
            "libsolv0-dev": "0.6.30-1ubuntu0.1~esm1",
            "libsolvext0": "0.6.30-1ubuntu0.1~esm1",
            "libsolvext0-dbgsym": "0.6.30-1ubuntu0.1~esm1",
            "libsolvext0-dev": "0.6.30-1ubuntu0.1~esm1",
            "python-solv": "0.6.30-1ubuntu0.1~esm1",
            "python-solv-dbgsym": "0.6.30-1ubuntu0.1~esm1",
            "python3-solv": "0.6.30-1ubuntu0.1~esm1",
            "python3-solv-dbgsym": "0.6.30-1ubuntu0.1~esm1"
          }
        ],
        "availability": "Available with Ubuntu Pro: https://ubuntu.com/pro"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://ubuntu.com/security/notices/USN-4851-1"
    },
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2018-20532"
    },
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2018-20533"
    },
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2018-20534"
    }
  ]
}